app/javascript/packages/document-capture/context/failed-capture-attempts.tsx

```diff
@@ -0,0 +1,102 @@
+import { createContext, useState } from 'react';
+import type { ReactNode } from 'react';
+import useCounter from '../hooks/use-counter';
+
+interface CaptureAttemptMetadata {
+  isAssessedAsGlare: boolean;
+  isAssessedAsBlurry: boolean;
+}
+
+interface FailedCaptureAttemptsContextInterface {
```

Comment:
I don't have a strong stance either way, and I know there are [some conventions incorporating "Interface" in the name](https://softwareengineering.stackexchange.com/questions/117348/should-interface-names-begin-with-an-i-prefix), but elsewhere I'd been tending to suffix "Value" for the interface describing the context value shape. I think we should aim to be consistent whichever way we want to go with the naming.

https://github.com/18F/identity-idp/blob/ba74e80660ea360f51afd635b259f72087f2ee0e/app/javascript/packages/document-capture/context/marketing-site.tsx#L16

https://github.com/18F/identity-idp/blob/ba74e80660ea360f51afd635b259f72087f2ee0e/app/javascript/packages/form-steps/form-steps-context.tsx#L3

https://github.com/18F/identity-idp/blob/ba74e80660ea360f51afd635b259f72087f2ee0e/app/javascript/packages/verify-flow/context/flow-context.tsx#L3